dkron/agent.go

```diff
@@ -16,8 +17,10 @@ import (
 	"time"
 
 	metrics "github.com/armon/go-metrics"
+	"github.com/devopsfaith/krakend-usage/client"
```

Comment:
Use of an analytics client of a lesser known third party is a (privacy) risk. Might be better to fix this to a version we have verified to be functioning as intended.
Or we could include it entirely; it is a simple module and it seems to me it would need little maintenance.

Reply:
I understand your concern, but if you look at the module you will see the last PR is mine, this is because I know those guys and I fully trust the module, we can use a fork though, but I think it's not necessary, and the module will be checked anyway for API changes on each update. On the other side, the server, that it's controlled by me, will not accept anything outside the predefined current fields.

Reply:
I looked through the module during review to see what it was doing and I'm fine with it as it is now - otherwise I would have protested a lot more :) I'm not concerned about the server side.

The issue is the trusting part. I trust you, you trust them. But should users trust us and our judgement?

I would be more comfortable if the version was fixed to the current commit.
And to handle it completely correctly, the module would have to be reviewed each time it needs to be upgraded. Considering the module has been unchanged for two years before your PR, I don't expect a lot of changes. So I think that's acceptable.